### PR TITLE
Register new package DiffEqCallbacks.jl v0.0.1

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaDiffEq/DiffEqCallbacks.jl](https://github.com/JuliaDiffEq/DiffEqCallbacks.jl)
Release: [v0.0.1](https://github.com/JuliaDiffEq/DiffEqCallbacks.jl/releases/tag/v0.0.1)
Travis: [Travis Build Status](https://travis-ci.org/JuliaDiffEq/DiffEqCallbacks.jl/builds/191098203)
cc: @ChrisRackauckas

### CODE REVIEW

**Comment by dpsanders** [comment]
Does such a small account of code really need to be a new package? Can't it just go eg in DiffEqBase?

**Comment by ChrisRackauckas** [comment]
I want to, with the release of the integrator and callback interfaces, keep DiffEqBase stable. It has the interface components, they should be mostly solid now, they shouldn't move much anymore. There will be a change which @mauro3 is helping me with which is to setup the promotions of different problems to another, and the addition of IMEX problems (+ expanding choices for DAEs through them), but that's it. It is setup to hold problem/solution definitions and their interfaces and should be a done deal with no more touching after a little bit more.

That said, these callbacks will be able to be applied everywhere. I have a list of things I am planning to add:

https://github.com/JuliaDiffEq/DiffEqCallbacks.jl/releases/tag/v0.0.1

Adding some of this will add dependencies, so it doesn't make sense in DiffEqBase (and there's no reason to bloat the Base with this kind of stuff). But there's also no good catchall for these other than DifferentialEquations.jl, but DifferentialEquations.jl is simply a metapackage (with the default algorithm choices, but that's because it's the only place where everything is a dependency). So it makes sense for it to be its own library. 

I note in the release that this a very early release for this package. Mostly because it was a user request (https://github.com/JuliaDiffEq/DifferentialEquations.jl/issues/118), but also because right now I am "re-stabilizing", tagging everything that works well in its current form, and will be letting all of these changes sit for awhile so that way users can experience a stable product:

https://github.com/JuliaDiffEq/DifferentialEquations.jl/issues/130

I could leave this one part out of that "stable ecosystem release", but the only reason to do so would be a sense of purity in the idea of "what is a package". Since it's a name I don't think others will be wanting anyways, I don't think there's a reason to wait and cause a feature disruption until it fills up more.

I hope that explains my intentions.

**Comment by dpsanders** [comment]
OK thanks.

File: DiffEqCallbacks/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.5
+DiffEqBase
```
Comment (tkelman) [inline DiffEqCallbacks/versions/0.0.1/requires]: needs a minimum version for `DiscreteCallback`

**Pull request merged**